images/libvirt-kubevirt/libvirtd.sh

```diff
@@ -63,4 +63,8 @@ fi
 
 echo "cgroup_controllers = [ ]" >> /etc/libvirt/qemu.conf
 
-/usr/sbin/libvirtd -l
+if [[ -n "$LIBVIRTD_DISABLE_TCP" ]]; then
```

Comment:
That looks like a nice config switch for kubevirt/libvirt. BTW. I was not aware that we have a duplicated version of libvirtd.sh here too.